templates/taxi/driver_list.html

```diff
@@ -1,30 +1,41 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
-    </h1>
+  <h1>Driver List
+    <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
+      Create
+    </a>
+  </h1>
```

Comment:
The `<h1>` tag is missing a closing tag. Ensure to close the `<h1>` tag properly to maintain valid HTML structure.